content/glance.pl.md

```diff
@@ -0,0 +1,130 @@
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+
+title: "ATAG w skrócie"
+nav_title: "ATAG w skrócie"
+lang: pl   # Change "en" to the translated language shortcode
+last_updated: 2024-03-05   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+translators:
+  - name: "Stefan Wajda"
+
+
+github:
+  repository: w3c/wai-intro-atag
+  path: content/glance.pl.md   # Add the language shortcode to the middle of the filename, for example: content/glance.fr.md
+  
+  
+permalink: /standards-guidelines/atag/glance/pl   
+ref: /standards-guidelines/atag/glance/   # Do not change this
+
+layout: default
+feedbackmail: wai@w3.org
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words, including "Date:" and "Editors:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: |
+  <p><strong>Data:</strong> Uakualniono 18 maja 2016.</p>
+  <p><strong>Redaktorzy:</strong> Jeanne Spellman i <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Opracowane z udziałem Grupy Roboczej ds. dostępności narzędzi autorskich (<a href="https://www.w3.org/WAI/AU/">AUWG</a>) oraz Grupy Roboczej ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>).</p>
+
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Podsumowanie" class="full" %}
+{:/}
+
+Ta strona zawiera sparafrazowane podsumowanie Wytycznych dla dostępności narzędzi autorskich (ATAG) 2.0. Normatywna specyfikacja techniczna znajduje się na stronie
+[www.w3.org/TR/ATAG20](https://www.w3.org/TR/ATAG20/).
+
+**Aby dowiedzieć się więcej o ATAG 2.0, zapoznaj się z [[Omówieniem ATAG]](/standards-guidelines/atag/pl
+).**
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+
+## [Część A](https://www.w3.org/TR/ATAG20/#part_a): Twórz dostępne interfejsy narzędzi autorskich {#PartA}
+
+{::nomarkdown}
+{% include box.html type="start" title='<a href="https://www.w3.org/TR/ATAG20/#principle_a1">A1</a> Interfejs użytkownika narzędzia autorskiego jest zgodny z obowiązującymi wytycznymi dla dostępności' h=3 class="" %}
+{:/}
+
+-   Zapewnij dostępność funkcjonalności opartych na interfejsach internetowych.
+-   Zapewnij dostępność funkcjonalności opartych na interfejsach nieinternetowych.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{% include box.html type="start" title='<a href="https://www.w3.org/TR/ATAG20/#principle_a2">A2</a> <a href="https://www.w3.org/TR/ATAG20/#def-Editing-View">Widoki redaktorskie</a> są postrzegalne' h=3 class="" %}
+{:/}
+
+-   Zapewnij twórcom treści alternatywne.
+-   Zapewnij, aby prezentacja w widoku redaktorskim była [określona programowo](https://www.w3.org/TR/ATAG20/#def-Programmatically-Determined)]
```

Comment:
>  Zapewnij, aby prezentacja w widoku redaktorskim była określona programowo

@zwiastunsw Which of these 2 meanings is it closest to?
1. Ensure presentation in editing-view is programmatically determined
2. Ensure presentation in editing-view can be programmatically determined

Reply:
changed in https://github.com/w3c/wai-intro-atag/pull/66/commits/70eff362aa6556e25cc348bc8338936676fa43ff